ag-grid-webcomponent/date_range_filter.js

```diff
@@ -0,0 +1,105 @@
+class DateRangeFilter {
+    init(params) {
+        this.params = params;
+        this.eGui = document.createElement('div');
+        this.eGui.style = `
+            padding:5px;
+        `
+        this.eGui.innerHTML = `
+            <div class="ag-filter-body-wrapper ag-simple-filter-body-wrapper" ref="eFilterBody" style="max-height: 969px;">
```

Comment:
I don't like this custom view block. Especially using AgGrid classes that might be removed in future updates and we might not realize that it breaks the filter UI.